src/Form/Question.php

```diff
@@ -136,6 +136,14 @@ public function prepareInputForUpdate($input)
     private function prepareInput(&$input)
     {
         $question_type = $this->getQuestionType();
+
+        if (
+            isset($input['default_value'])
+            && is_array($input['default_value'])
+        ) {
+            $input['default_value'] = implode(',', $input['default_value']);
+        }
+
```

Comment:
Since the default value field may be a direct value or an array depending on the question type, we could add a "getDefaultValueFormat()" method to the `QuestionType` interface.

Then, we can call this method to be sure of the expected default value format (and allow us to support more formats if needed in the future).